### DISCUSSION TITLE
filtering multiple same processes to single process

### DISCUSSION THREAD

**Discussion by vuslatx** [winsiderss/systeminformer#1091] [Questions / Answers] [answered]
Firstly, excuse me for my bad English. Thanks for good app.

I don't want to see multiple same process. For example, there are a lot of svchost.exe, firefox.exe, explorer.exe. I want to see them singularly.

Is there a way to do that?

Not: I use nightly build. It is 3.0.4486

**Comment by Biswa96** [comment, marked as answer]
That can be done by clicking on the small down arrow, like this:

![image](https://user-images.githubusercontent.com/31443074/149090485-78d9f43a-ef10-44aa-bae4-d626bbc3eb1f.png)

**Reply by vuslatx** [reply]
I can't see small down arrow.
![9931](https://user-images.githubusercontent.com/65629417/149094239-65f3a088-d24f-46c0-8d26-8fcb44d2ce5b.png)

**Reply by Biswa96** [reply]
Click on the "Name" tab at the upper left corner two or three times. The column will change to tree view.

**Reply by vuslatx** [reply]
Thanks. It works.
